lib/osf-components/addon/components/resources-list/resource-card/styles.scss

```diff
@@ -0,0 +1,25 @@
+.resourceItem {
+    padding: 12px 0;
+    border-top: solid 1px $color-border-gray;
+    min-height: 60px;
+}
+
+.resourceType {
+    display: inline-flex;
+    font-weight: bold;
+}
+
+.icon {
+    width: 18px;
+    height: 20px;
+    background-size: contain;
+    margin: 0 6px;
+}
+
+.data {
+    background-image: url('engines-dist/registries/assets/img/badges/data_small_color.png');
```

Comment:
I'm starting to think we should move the logos from registries assets to the osf-assets submodule, since the badge is not limited for use in registries.

Reply:
We're not using them outside of registries yet, though, are we?

Reply:
Do we use them on `node-card`s?

Reply:
The registration tab on the project page and the my-registrations page both use `node-card` component to show registrations. They shouldn't show badges for projects. But if the assets are referenced by components outside of the `registries` engine, should we move it out of the engine?

Reply:
@adlius is correct, these cards are used outside of the engine for the node's registrations page. I'll make a quick PR to move them so we all have them.